no-nuxt-pls/components/NewsItem/NewsItem.vue

```diff
@@ -2,13 +2,32 @@
   <div>
     <h2>{{news.title}} ({{news.votes}})</h2>
     <br>
-    <button @click="updateNews(1)">Upvote</button>
-    <button @click="updateNews(-1)">Downvote</button>
-    <button @click="deleteNews">Remove</button>
+    <button v-if="token" @click="updateNews(1)">Upvote</button>
+    <button v-if="token" @click="updateNews(-1)">Downvote</button>
+    <button v-if="authorId === news.author.id" @click="deleteNews">Delete</button>
+    <button v-if="authorId === news.author.id">Edit</button>
```

Comment:
star For a delete and edit button that is only visible to the author of the post.